tornadox/blocks/sde/ibm.py

```diff
@@ -0,0 +1,58 @@
+"""Integrated Brownian motion."""
+
+from functools import partial
+
+import jax.lax
+import jax.numpy as jnp
+
+
+@partial(jax.jit, static_argnames=("num_derivatives",))
+def system_matrices_1d(*, num_derivatives):
```

Comment:
yes. I am a big fan of keyword-only arguments.

Reply:
Seems reasonable, but I think I'm less fan of it than you are. I see the point for larger functions with more arguments, e.g. the attempt step stuff, but here since there is only a single one I'm a bit more sceptic. I think here I would tend towards having this a normal python function argument. But feel free to decide either way and resolve this as you see appropriate :)